control-plane/csi-driver/controller/src/controller.rs

```diff
@@ -133,13 +132,9 @@ impl VolumeTopologyMapper {
     }
 
     /// Determine the list of nodes where the workload can be placed.
-    /// If volume is created as pinned (i.e. local=true), then the nexus and the workload
-    /// must be placed on the same node, which in fact means running workloads only on IO Engine
-    /// daemonset nodes.
-    /// For non-pinned volumes, workload can be put on any node in the cluster.
     fn volume_accessible_topology(&self) -> Vec<CsiTopology> {
         vec![rpc::csi::Topology {
-            segments: CsiControllerConfig::get_config().io_engine_selector(),
+            segments: CsiControllerConfig::get_config().csi_node_selector(),
```

Comment:
Question: Is it a good idea to bind csi-nodes with some lables as in here, or let it be anything?

Reply:
We can't bind this otherwise e2e will likely break - this will have to be discussed with a wider audience.
I would make it optional via cmdline and by default it should not bind.

Reply:
Sure.

Reply:
Removed it for now.